CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+* Any new project repositories must be approved by the Technical Steering Committee.
+
+## Submitting Changes
+Inbound contributions are done through [pull requests](https://github.com/open-ness/edgeapps/pulls) which include code changes, enhancements, bug fixes or new applications/features. If you are getting started, you may refer to Github's [how-to](https://help.github.com/articles/using-pull-requests/). With your contributions, we expect that you:
+
+* certify that you wrote and/or have the right to submit the pull request,
+* agree with the [Developer Certificate of Origin](http://developercertificate.org/),
+* sign-off your contribution with `Signed-off-by` tag in the commit message(s)
+* comply with OpenNESS [licensing](#license),
+* pass all Continuous Integration & Continuous Delivery (CI/CD) tools' checks,
+* test and verify the proper behaviour of your code, and
+* accompany the contribution with good quality documentation.
+
+Signing off the contribution is performed by the command `git commit --signoff` which certifies that you wrote it or otherwise have the right to pass it on as an open-source contribution.
+
+Application contributors have the implicit commitment to continually support their applications in whenever a bug is discovered or a test is broken due to the publication of a new release.
+
+It is always admired when application contributors provide end-to-end acceptance test methods.
+
+Big contributions, ideas or controversial features should be discussed with the developers' community and with the Technical Steering Committee through a Request for Comments (RFC). This is an essential early step to bring everybody to a common level of understanding so that when the final implementation is pushed, there will be common consensus of the acceptability of the contribution and smooth integration of it to the mainline. Requests for Comments should be submitted through a pull request prefixed with `[RFC]` in the title.
+
+## Contribution Acceptance Flow
+1. A contributor creates a pull request with their changes.
+2. Continuous Integration & Continuous Delivery (CI/CD) tools will be automatically triggered and their results are returned into the pull request.
+3. The contributor resolves all findings reported by CI/CD verifications.
+4. The contributor invites reviewers to the pull request and addresses their comments/feedback.
+5. Every pull request must receive 2 approvals in order to go forward with merging.
+6. The maintainer(s) must review and ensure that the change(s) introduced by the pull request meets the acceptance criteria.
+6. Merging the pull requests can only be performed by the maintainer(s).
+
+## How to report an issue/bug/enhancement
+It is encouraged to use the [GitHub Issues](https://github.com/open-ness/edgeapps/issues) tool to report any bug, issue, enhancement or to seek help.
+
+## Resources
```

Comment:
what will be the impact of Enhanced and Open on this? How do we make the ref. doc available ? 

Are we saying all the app developers will be Enhanced Openness customers?

Reply:
Yes. The intention is to bring-up contributors up-to-speed with the OpenNESS solution.